sqlgen/mysql.go

```diff
@@ -28,6 +28,10 @@ func (w *SimpleWhere) ToSQL() (string, []interface{}) {
 	return buffer.String(), w.Values
 }
 
+type SQLQuery interface {
```

Comment:
Nit: Could we rename this SQLStatement since we're using this interface for DeleteRow and InsertRow as well? (in my mind SQL query is synonymous with SQL select, though that may be inaccurate. Feel free to ignore if you disagree).